src/excmds.ts

```diff
@@ -5896,23 +5896,35 @@ async function js_helper(str: string[]) {
  *
  * Usage:
  *
- *        `js [-p] javascript code ... [arg]`
+ *     `js javascript code ...`
  *
- *        `js [-s|-r|-p] javascript_filename [arg]`
+ *     `js -p javascript code ... arg`
+ *
+ *     `js [-s|-r] javascript_filename`
+ *
+ *     `js -p [-s|-r] javascript_filename arg`
+ *
+ *     `js -d# [-s|-r] javascript_filename # arg1 arg2 ...`
+ *     (where `#` is any char)
```

Comment:
```suggestion
 *     `js -d³ [-s|-r] javascript_filename³ arg1 arg2 ...`
 *     (where `³` is any char  that you can guarantee won't appear in your JS code)
```

`#` works at the moment but i'm reluctant to encourage it because it's also our comment character